docs/web-apps/automated-testing/playwright/cucumber.md

```diff
@@ -0,0 +1,870 @@
+---
+id: cucumber
+title: Configuring Your Playwright with Cucumber Tests
```

Comment:
I think we should change the general wording to `Cucumber Tests with Playwright`.
A benign change, but main focus are the Cucumber tests, with playwright as the automation framework underneath.

Also worth mentioning somewhere that this is CucumberJs, specifically.